stmhal/Makefile

```diff
@@ -321,6 +321,16 @@ $(BUILD)/%_$(BOARD).c $(HEADER_BUILD)/%.h $(HEADER_BUILD)/%_af_const.h $(BUILD)/
 $(BUILD)/pins_$(BOARD).o: $(BUILD)/pins_$(BOARD).c
 	$(call compile_c)
 
+GEN_STMCONST_HDR = $(BUILD)/modstm_const.gen.h
```

Comment:
right but pins_qstr.h and modstm_qstr.h are just intermediate files which are passed into makeqstrda.py.

qstrdefs.h and qstrdefsport.h should definitely have .h extensions.

I'm happy either way (calling them .h or calling them .qstr)